nav2_costmap_2d/plugins/range_sensor_layer.cpp

```diff
@@ -292,8 +292,11 @@ void RangeSensorLayer::updateCostmap(
   in.header.stamp = range_message.header.stamp;
   in.header.frame_id = range_message.header.frame_id;
 
+  std::string * errstr = NULL;
```

Comment:
Might want to include that error string in the RCLCPP INFO when it fails to give more context, if you're going to collect it!